src/Microsoft.Health.Fhir.SqlServer.UnitTests/Features/Import/SqlServerBulkImportOperationTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Reflection;
+using Microsoft.Health.Fhir.SqlServer.Features.Schema.Model;
+using Microsoft.Health.Fhir.SqlServer.Features.Storage;
+using Microsoft.Health.SqlServer.Features.Schema.Model;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.SqlServer.UnitTests.Features.Import
+{
+    public class SqlServerBulkImportOperationTests
+    {
+        [Fact]
+        public void GivenResourceRelatedTables_WhenNewIndexesAdded_BulkImportOperationShouldSupportNewIndexes()
```

Comment:
>BulkImportOperationShouldSupportNewIndexes [](http://example.com/codeflow?start=67&length=42)

What about indexes added via Azure's Database Advisor

Reply:
Thanks for this comment, we do not aware this before. I think we can leave it for now since disable indexes now is not necessary, and for this test we want to block change for indexes and go through if we need to add the new indexes to this list. We would test this and maybe we can add document for this in advanced section of bulk import operation.